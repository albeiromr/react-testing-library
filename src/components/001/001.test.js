import { render, screen} from "@testing-library/react";
import Component001 from "./001";

test("renders hello world", () => {
  render(<Component001 />);
  const title = screen.getByText("hello world");
  expect(title).toBeVisible();
  expect(title).toMatchSnapshot("<h1>hello world</h1>");
  expect(title).not.toBeEmptyDOMElement();
});

test("renders hello world 2", () => {
    render(<Component001 />);
    const title = screen.getByText("I am Alex");
    expect(title).toBeInTheDocument();
    expect(title).toBeVisible();
});
