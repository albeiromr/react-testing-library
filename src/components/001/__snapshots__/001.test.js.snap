// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders hello world: <h1>hello world</h1> 1`] = `
<h1>
  hello world
</h1>
`;
